assets/src/domain/eventEditor/interfaces/types.ts

```diff
@@ -29,5 +29,6 @@ declare global {
 	interface Window {
 		eeEditorData: EEEditorData;
 		eejsdata: EEJSData;
+		eeDomain: string;
```

Comment:
noticed that this is in `eventEditor` domain (`assets/src/domain/eventEditor/interfaces/types.ts`),
this will be moved later?

Reply:
Yes, when global assets are combined, those 3 will be removed and replaced with that global object.